### PR TITLE
Create collection laravel libraries

### PR DESCRIPTION
**Pull request by ezhasyafaat**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: collections/laravel-awesome-library/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ - hammerstonedev/fast-paginate
+ - opcodesio/log-viewer
+ - devtical/laravel-helpers
+ - thephpleague/flysystem-aws-s3-v3
+ - getsentry/sentry-laravel
+display_name: Laravel Awesome Library
+created_by: ezhasyafaat
+image: laravel-logo.png
+---
+List of awesome libraries that can be used for Laravel applications.
```
Comment (MikeMcQuaid) [inline collections/laravel-awesome-library/index.md:17]: How about just 
```suggestion
Collection of useful libraries for Laravel applications. 
```

and renaming to `laravel-libraries`?

**Review by MikeMcQuaid — APPROVED** [review]

File: collections/laravel-library/index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ - laravel-mix/laravel-mix
+ - filamentphp/filament
+ - bavix/laravel-wallet
+ - mpociot/teamwork
+ - hammerstonedev/fast-paginate
+ - opcodesio/log-viewer
+ - devtical/laravel-helpers
+ - thephpleague/flysystem-aws-s3-v3
+ - getsentry/sentry-laravel
+display_name: Laravel Awesome Library
```
Comment (MikeMcQuaid) [inline collections/laravel-library/index.md:13]:
```suggestion
display_name: Laravel Libraries
```

and rename `laravel-library` to `laravel-libraries`

**Review by MikeMcQuaid — COMMENTED** [review]
Actually sorry one last bit!

**Comment by ezhasyafaat** [comment]
Thanks fot the advice

**Comment by ezhasyafaat** [comment]
Please check it again @MikeMcQuaid, thanks

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again @ezhasyafaat!

**Comment by ezhasyafaat** [comment]
Hi @MikeMcQuaid i've add new awesome library, please check it again, thank you

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks @ezhasyafaat!

**Pull request merged**